modules/beachfrontBidAdapter.js

```diff
@@ -13,7 +13,6 @@ import {BANNER, VIDEO} from '../src/mediaTypes.js';
 import { getFirstSize, getOsVersion, getVideoSizes, getBannerSizes, isConnectedTV, getDoNotTrack, isMobile, isBannerBid, isVideoBid, getBannerBidFloor, getVideoBidFloor, getVideoTargetingParams, getTopWindowLocation } from '../libraries/advangUtils/index.js';
 
 const ADAPTER_VERSION = '1.21';
-const GVLID = 335;
```

Comment:
it seems we have a new gvl here https://www.adexchanger.com/streaming/seedtag-acquires-beachfront-for-deeper-roots-in-tv-and-streaming/ ?